packages/components/src/Checkbox/Checkbox.css

```diff
@@ -66,6 +72,14 @@
 }
 
 .label {
-  display: inline-block;
   margin: 0 var(--space-small);
 }
+
+.description {
+  margin-top: var(--space-smaller);
+  margin-left: var(--space-large);
+}
+
+.description > p {
```

Comment:
I have added a comment in an upcoming backlog ticket to replace this with an updated version of `Text`